diff --git a/src/Process.js b/src/Process.js
index 7f1d7c662aabade3c75e69ec2fee9f3175e9fd96..254a379180ceaa2d7fb2662d4035791405e9c1ef 100644
--- a/src/Process.js
+++ b/src/Process.js
@@ -289,17 +289,17 @@ class Process {
         const previewLink = await this.getPreviewLink(2280);
         const url = `${previewLink.url.replace(/^http/, 'ws')}/process/session/${sessionId}/command/${commandId}/logs?follow=true`;
         let ws;
-        if (Runtime_1.RUNTIME === Runtime_1.Runtime.BROWSER) {
+        // if (Runtime_1.RUNTIME === Runtime_1.Runtime.BROWSER) {
             ws = new isomorphic_ws_1.default(url + '&DAYTONA_SANDBOX_AUTH_KEY=' + previewLink.token, `X-Daytona-SDK-Version~${this.clientConfig.baseOptions.headers['X-Daytona-SDK-Version']}`);
-        }
-        else {
-            ws = new isomorphic_ws_1.default(url, {
-                headers: {
-                    ...this.clientConfig.baseOptions.headers,
-                    'X-Daytona-Preview-Token': previewLink.token,
-                },
-            });
-        }
+        // }
+        // else {
+        //     ws = new isomorphic_ws_1.default(url, {
+        //         headers: {
+        //             ...this.clientConfig.baseOptions.headers,
+        //             'X-Daytona-Preview-Token': previewLink.token,
+        //         },
+        //     });
+        // }
         await (0, Stream_1.stdDemuxStream)(ws, onStdout, onStderr);
     }
     /**
